desktop/apps/auction2/works.js

```diff
@@ -2,11 +2,14 @@ import React from 'react'
 import ArtworksContainer from './components/auction_artworks'
 import HeaderContainer from './components/header'
 
-const Works = () => (
+export const Works = () => (
   <div>
-    <HeaderContainer />
     <ArtworksContainer />
   </div>
 )
 
-export default Works
+export const Header = () => (
+  <div>
+    <HeaderContainer />
+  </div>
+)
```

Comment:
Just realized that these are wrapping around an existing wrapper component-- probably a relic of me trying to figure things out and wrap everything in a _single_ container. I think I'll just import `ArtworksContainer` and `HeaderContainer` directly in client.js